### DISCUSSION TITLE
Proscenic M6 Pro questions

### DISCUSSION THREAD

**Discussion by dnalor** [Hypfer/Valetudo#2157] [Q&A / Support] [answered]
### Are you already using Valetudo?

No, but planning to do it in the near future

### Your questions

I found a used Proscenic M6 Pro and would like to buy it and install Valetudo.
Before I do so, I would appreciate it if someone could confirm the following two points:

For the Proscenic M6 Pro, the docs say:
>Because these robots use a non-miio cloud implementation, getting them to work with Valetudo means reflashing them to a Viomi V6. That’s possible, because the hardware is exactly the same.

However, I haven't found a discription of how to do that.  
Am I therefore correct in assuming that this will simply be a "side effect" of flashing the DustBuilder image?

On the Valetudo site, the Proscenic M6 Pro is listed as "supported"  
DustBuilder, however, considers it "legacy":
>Legacy supported devices. Not recommend to purchase as new devices. Are rootable and supported by Valetudo, but wont receive firmware updates anymore

Am I therefore correct in assuming that, while it won't receive firmware updates, it will receive Valetudo updates?

Thanks

**Comment by Hypfer** [comment, marked as answer]
> Am I therefore correct in assuming that this will simply be a "side effect" of flashing the DustBuilder image?

Yes

> Am I therefore correct in assuming that, while it won't receive firmware updates, it will receive Valetudo updates?

Yes

**Reply by dnalor** [reply]
Thanks for the quick answer!
